src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CompilationContext.cs

```diff
@@ -100,70 +94,76 @@ internal sealed class CompilationContext
                 _displayClassVariables.Values.Any(v => v.Kind == DisplayClassVariableKind.This));
         }
 
-        internal CommonPEModuleBuilder CompileExpressions(
```

Comment:
In the future I think it's better to separate annotations and refactorings. Reviews of annotation only changes goes much faster and don't have to add in the complexity of semantic changes on top.

Reply:
I agree and tried to limit the changes only to such that were necessary to make the annotations more reasonable. I guess I could disable the annotations in some regions for first pass and fix that code in a follow up.